src/pages/references/node-configuration.md

```diff
@@ -0,0 +1,477 @@
+---
+title: Node Configuration
+description: Configuration parameters and options for the stacks-node binary
+icon: TestnetIcon
+images:
+  large: /images/pages/testnet.svg
+  sm: /images/pages/testnet-sm.svg
+---
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [Subcommands](#subcommands)
+- [Configuration File Options](#configuration-file-options)
+  - [Section: node](#section-node)
+  - [Section: events_observer (optional)](#section-events_observer-optional)
+  - [Section: connection_options (optional)](#section-connection_options-optional)
+  - [Section: burnchain](#section-burnchain)
+  - [Section: mstx_balance](#section-mstx_balance)
+
+## Installation
+
+See this page for installation instructions.
+[@page-reference | inline]
+| /stacks-blockchain/running-testnet-node
+
+## Usage
+
+```bash
+stacks-node sub-command [--subcommand-option <value>]
+```
+
+## Subcommands
+
+### mocknet
+
+Start a node based on a fast local setup emulating a burnchain. Ideal for smart contract development.
+
+Example:
+
+```bash
+stacks-node mocknet
+```
+
+### krypton (deprecated)
```

Comment:
I intentionally left out the `helium` and `argon` subcommands since we're past those testnet phases, and put `deprecated` for the `krypton` subcommand since this testnet phase will be completed soon. Was this the right move?

Reply:
I assume that it will be possible to configure the testnet xenon to run with a single node for testing.

Reply:
I don't have the full understanding of the config options but my general suggestion would be to not assume folks know what phase we're currently in. So the heading based on the phases would likely be very confusing. Maybe we can add the deprecation node to the specific commands/options? 

To answer the other question - deprecated content (assuming it is not relevant anymore) shouldn't be added to the docs. would rather increase confusion

Reply:
Even though we're now past some phases, these are actual subcommands to the `stacks-node` binary. I think we just haven't had the time to remove them from code yet. So while I could remove them from the docs, users will still see them appear in the `stacks-node`'s help message until it's removed in code.

I agree on the deprecation point, just didn't want this to come across as "hey the PBC is hiding info!", and instead just offer the docs based on the current status of the `stacks-node` binary.

Reply:
that makes sense! For usability reasons, I'd focus on documenting what we'd like folks to use as opposed to everything there is. Maybe we could add a note to the references explicitly saying that the binary includes other commands that are deprecated but not yet removed:

```
-> Note that the `stacks-node` binary could include deprecated commands that are not documented on this page. Deprecated commands could be accessible until they are fully removed from the sources
```